src/test/java/com/google/api/codegen/testdata/python_doc_docs_conf_library.baseline

```diff
@@ -0,0 +1,140 @@
+============== file: docs/conf.py ==============
+
+import sys
+import os
+import shlex
+
+sys.path.insert(0, os.path.abspath('..'))
+
+__version__ = '0.14.0'
+
+
+
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.coverage',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.viewcode',
+]
+
+autoclass_content = 'both'
+autodoc_default_flags = ['members']
+autosummary_generate = True
+
+
+templates_path = ['_templates']
+
+source_suffix = '.rst'
+
+
+master_doc = 'index'
+
+project = u'google-cloud-library-v1'
+copyright = u'2016, Google'
+author = u'Google APIs'
+
+release = __version__
+version = '.'.join(release.split('.')[0:2])
+
+language = None
+
+
+exclude_patterns = ['_build']
+
+
+
+
+
+pygments_style = 'sphinx'
+
+
+
+todo_include_todos = True
+
+
+
+html_theme = 'sphinx_rtd_theme'
+
+
+
```

Comment:
Ah, whoops. The `#` are comments in SnippetSet templates but not in mustache. I will fix this.